wpl_cli/src/group/cmd.rs

```diff
@@ -0,0 +1,40 @@
+use super::create::{run as create_group_account, CreateArgs};
+use super::update::{run as update_group_account, UpdateArgs};
+
+use anyhow::Result;
+use clap::{Args, Subcommand};
+use solana_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::signature::Keypair;
+
+#[derive(Debug, Clone, Args)]
+pub struct GroupSubCommand {
+    #[clap(subcommand)]
+    pub action: Commands,
+}
+
+#[derive(Debug, Clone, Subcommand)]
+pub enum Commands {
+    #[clap(name = "create")]
+    /// Create a new group account (collection)
+    Create(CreateArgs),
+    #[clap(name = "update")]
+    /// Update a group account (collection)
+    Update(UpdateArgs),
+}
+
+pub async fn subcommand(
+    async_client: RpcClient,
```

Comment:
```suggestion
    client: RpcClient,
```

Reply:
The client and keypair for a context object.

```rust
struct Context {
  client: Rpc,
  keypair: Keypair
}
```

Then all subcommands take the context which is shared resources for all subcommands and args for specific settings.